### ISSUE TITLE
verification and test results

### ISSUE THREAD

**Issue by Z-Lee-corder** [blakechen97/SASA#6]

**Comment by blakechen97** [comment]
Hi,

I hope you are using the same experimental setting to ours. It seems that your result is from the 3DSSD baseline. To the best of our knowledge, the performance on val split for 3DSSD baseline is around 83 mAP while that for 3DSSD with SASA is usually above 85 mAP (evaluated under 40 recall points). Please make sure that the SASA side loss and S-FPS are both enabled. Besides, please note that if you have changed the batch size, you need to scale the learning rate accordingly.

When submitting to the KITTI test server, we follow the strategy from PV-RCNN to randomly divide the entire labeled training set and use 80% samples for training and 20% samples for validation (as mentioned in the paper). 

Please feel free to contact me if you have any questions.

**Comment by Z-Lee-corder** [comment]
I got this result by training with 3dssd_sasa.yaml.   And the SASA side loss and S-FPS are both enabled. I only changed the batch size to 4, and the rest remained unchanged. I wonder if the matching setting of batch size and learning rate, and the size division of training set and verification set will have such a great impact on the experimental results?  （about 3% mAP)

**Comment by blakechen97** [comment]
Learning rate is a very important factor. If you are using a total batch size of 4, it is recommended to set the learning rate around `0.0025`.

**Comment by Z-Lee-corder** [comment]
Thanks, author,  I have used the learning rate  0.0025 and batch size 4, and the result of 3dssd_sasa has achieved 84%(AP10) and 85%(AP40) in kitti val set.   But I found 3dssd baseline sometimes has also this result (  84%(AP10)  and  85%(AP40)  ) by using learning rate  0.0025 and batch size 4.
I don't understand why？
